data/types/device/alarm.go

```diff
@@ -15,10 +15,15 @@ func init() {
 
 type Alarm struct {
 	AlarmType *string `json:"alarmType" bson:"alarmType" valid:"devicealarmtype"`
+	Status    *string `json:"status,omitempty" bson:"status,omitempty" valid:"-"`
```

Comment:
its a requirement for validators to have it first and the other tags it is the opposite